python/vineyard/core/tests/test_client.py

```diff
@@ -281,3 +282,14 @@ def start_requests(rs, state, ipc_socket):
         r, message = rs.get(block=True)
         if not r:
             pytest.fail(message)
+
+
+def test_vineyard_read_and_write(vineyard_ipc_sockets):
+    path = os.path.dirname(vineyard_ipc_sockets[0])
+    # create a soft link of vineyard socket
+    os.symlink(vineyard_ipc_sockets[0], os.path.join(path, "vineyard.sock"))
+
+    value = (1, 2, 3, 4)
+    vineyard.write(value, path)
+    result = vineyard.read(path)
+    assert result == value
```

Comment:
These APIs not should be tested using the CSI driver, not here.

As without the CSI driver, we never expect user have such path where both name and ipc socket location is embeded.

Reply:
Make sense.